src/main/resources/db/migration/V4__create_role.sql

```diff
@@ -0,0 +1,18 @@
+CREATE SEQUENCE role_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+CREATE TABLE role (
+    id bigint DEFAULT nextval('role_id_seq'::regclass) NOT NULL,
+    name character varying(255) NOT NULL,
+    pairing_board_id bigint
+);
+
+ALTER TABLE ONLY role
+    ADD CONSTRAINT role_pkey PRIMARY KEY (id);
+
+
```

Comment:
Hey, ran this app locally and noticed that this migration doesn't actually make the foreign key between `role::pairing_board_id` and `pairing_board::id`.  Here's what I think you should add to get that in there:

```
ALTER TABLE ONLY role
    ADD CONSTRAINT pairing_board_id_fk FOREIGN KEY (pairing_board_id) REFERENCES pairing_board(id);
```

Also, since roles can never not be connected to a pairing_board, the `pairing_board_id` column should be marked as `NOT NULL`.

Lastly, here one thing the postgres dump did that I like and think should also be added 😛 

```
ALTER SEQUENCE role_id_seq OWNED BY role.id;
```

(I think that causes the sequence to be automatically removed if the column is removed.)